core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -79,9 +79,7 @@ contract Liquidatable is PricelessPositionManager {
     */
 
     // Liquidations are unique by ID per sponsor
-    mapping(address => mapping(uint => LiquidationData)) public liquidations;
-    // Keeps track of last used liquidation ID per sponsor
-    mapping(address => uint) public sponsorLiquidationIndex;
+    mapping(address => LiquidationData[]) public liquidations;
```

Comment:
What effect does this change have on the running time of `delete liquidations[sponsor][id]` on line 352?

Reply:
Good point -- thanks for pointing this out.

I just looked into the [solidity docs on this](https://solidity.readthedocs.io/en/v0.5.13/types.html#delete), and by my reading, this should do what we expect/want: zero out the struct without touching the length of the array.

Reply:
I think you are right based on the line 

> It is important to note that delete a really behaves like an assignment to a, i.e. it stores a new object in a

[Another reference to this situation](https://ethereum.stackexchange.com/questions/1527/how-to-delete-an-element-at-a-certain-index-in-an-array)